test/e2e/tests/test_negative_security.py

```
@@ -429,3 +429,48 @@ def test_special_characters_in_subscription_header(self):
+
+    The HTTPRoute blocks /maas-api/internal/* at the gateway level so that
+    requests never reach the maas-api backend. In-cluster callers (Authorino,
+    cleanup CronJob) access these endpoints directly via the cluster-local
+    Service, bypassing the gateway.
+    """
+
+    _INTERNAL_PATHS = [
+        "/internal/v1/api-keys/validate",
+        "/internal/v1/api-keys/cleanup",
+        "/internal/v1/subscriptions/select",
+    ]
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)